security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+Furthermore, if you notice that the person walks into the office of Company X using an electronic badge, the anonymity set shrinks to the number of Company X's employees and visitors.
+Finally, you may notice the license number of the car they used to arrive at the place.
+If you are a casual observer, this doesn't give you much.
+However, if you are a city official and have access to the database that matches license plate numbers with names, you can narrow down the anonymity set to just a few people: the car owner and their closest friends and relatives.
+
+This example illustrates a few important points.
+First, every bit of information may bring the adversary closer to their goal.
+It may not be necessary to shrink the anonymity set to the size of one.
+For instance, if the adversary plans a targeted DoS attack and can take down 100 servers, the anonymity set of 100 suffices.
+Second, the adversary can cross-correlate information from different sources.
+Even if a privacy leak looks relatively benign, we never know what it can achieve in combination with other data sources.
+Finally, especially in cryptographic settings, the attacker always has the "last resort" of brute force search.
+Cryptographic primitives are designed so that it is practically impossible to guess a secret such as a private key.
+Nevertheless, each bit of information brings the adversary closer to this goal, and at some point, it becomes attainable.
+
+In terms of Lightning, deanonymizing generally means deriving a correspondence between payments and users identified by node IDs.
+Each payment may be assigned a sender anonymity set and a receiver anonymity set.
+Ideally, the anonymity set consists of all the users of the network.
+This assures that the attacker has no information whatsoever.
+However, the real network leaks information that allows an attacker to narrow down the search.
+The smaller the anonymity set, the higher the chance of successful deanonymization.
+
+
+=== Differences between LN and Bitcoin in terms of privacy
+
+Bitcoin was initially perceived as an anonymous currency.
+Indeed, Bitcoin addresses are not linked to real identities.
+However, all transactions are broadcast in cleartext and can be analyzed.
+Multiple companies deanonymize users of Bitcoin and other cryptocurrencies.
+
+At first glance, Lightning provides better privacy than Bitcoin because Lightning payments are not broadcast to the whole network.
+While this improves the privacy baseline, other properties of the Lightning protocol may make anonymous payments more challenging.
+For instance, larger payments have fewer routing options.
+This may allow an adversary who controls well-capitalized nodes to route most large payments and discover payment amounts and probably other details.
+Another relevant difference between Lightning and Bitcoin is that Lightning nodes maintain a permanent identity, whereas Bitcoin nodes do not.
```

Comment:
Although Bitcoin nodes do not assume a permanent identity, wallet clustering achieves similar outcomes for wallets without/under utilized/or ineffective privacy techniques.